draft-ietf-mls-protocol.md

```diff
@@ -3797,14 +3801,18 @@ The full tree can be included via the `ratchet_tree` extension
 The `signature` on the GroupInfo struct authenticates the HPKE public key, so
 that the joiner knows that the public key was provided by a member of the group.
 The fields that are not signed are included in the key schedule via the
-GroupContext object. If the joiner is provided an inaccurate data for these
+GroupContext object. If the joiner has provided inaccurate data for these
```

Comment:
This is wrong, because we're talking about GroupInfo that the joiner consumes.  But this whole paragraph is wrong; will remove in a subsequent commit.